mitodl/ocw-hugo-themes
# getJSON replaced with resources.GetRemote

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-themes/issues/703

#### What's this PR do?
- Replaces all remote calls via `getJSON` with `resources.GetRemote` so we have more flexibility in error handling.

#### How should this be manually tested?
- Checkout this branch
- Open all those pages in which this change has been implemented and verify that those pages are rendering/showing content correctly/like before and there is no apparent change in the content.
